src/sentry/integrations/vsts/integration.py

```diff
@@ -0,0 +1,163 @@
+from __future__ import absolute_import
+from sentry import http
+
+from django import forms
+from sentry.web.helpers import render_to_response
+from sentry.integrations import Integration, IntegrationMetadata
+from sentry.pipeline import NestedPipelineView, PipelineView
+from sentry.identity.pipeline import IdentityProviderPipeline
+from sentry.utils.http import absolute_uri
+DESCRIPTION = """
```

Comment:
Doesn't the formatter complain about missing spaces after the imports?

Reply:
No I haven't gotten any complaints.